toolkit-maven-plugin/pom.xml

```diff
@@ -103,9 +103,16 @@
     </dependency>
 
     <dependency>
-      <groupId>org.mockito</groupId>
```

Comment:
Could you tell the reason why you switch from mockito to powermock?

Reply:
use powermock to test static method, mockito can not do it